draft-ietf-quic-qlog-quic-events.md

```diff
@@ -655,10 +655,18 @@ Most of these settings are typically set once and never change. However, they
 are usually set at different times during the connection, so there will
 regularly be several instances of this event with different fields set.
 
-Note that some settings have two variations (one set locally, one requested by the
-remote peer). This is reflected in the `owner` field. As such, this field MUST be
-correct for all settings included a single event instance. If you need to log
-settings from two sides, you MUST emit two separate event instances.
+Note that some settings have two variations (one set locally, one requested by
+the remote peer). This is reflected in the `owner` field. As such, this field
+MUST be correct for all settings included a single event instance. If you need
+to log settings from two sides, you MUST emit two separate event instances. If
```

Comment:
The use of "you" in this paragraph sticks out. There's nothing else like it in our qlog specs. I know it was in the old text but while we are here lets change it. Something like below perhaps

```suggestion
MUST be correct for all settings included a single event instance. If the settings
from two sides are required, they MUST be logged as two separate event instances. If
```